ui/src/components/Organization/List.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames/bind';
+import style from 'src/components/Organization/style.module.scss';
+
+const c = classNames.bind(style);
+
+interface Organizations {
+  id: string;
+  name: string;
+  banner?: string;
+  square_logo?: string;
+}
```

Comment:
1. Name `Organizations` implies plural, i.e this is a collection, but it is not
2. We already have `Organization` type defined in `entities/Organization`. Please import that type instead of duplicating the type definition